### PR TITLE
init fleet plugin install flagger and public testloader

### PR DESCRIPTION
**Pull request by LiZhenCheng9527**
**What type of PR is this?**
/kind feature
**What this PR does / why we need it**:
Complete install flagger and public testlaoder via fleet plugin
**Which issue(s) this PR fixes**:
Fixes # #435

### CODE REVIEW

File: pkg/fleet-manager/manifests/plugins/testloader.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+type: default
+repo: https://flagger.app
+name: loadtester
+version: 0.30.0
+targetNamespace: istio-system
```
Comment (Xieql) [inline pkg/fleet-manager/manifests/plugins/testloader.yaml:5]: miss a blank line

Comment (LiZhenCheng9527) [reply pkg/fleet-manager/manifests/plugins/testloader.yaml]: My pre-push file had a blank line, but the black line miss after push. And I compare other yaml file, find they all miss black lind in the end of file. So I think, it is GITHUB hands it

File: pkg/fleet-manager/fleet_plugin_flagger.go [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+			flaggerResources, err := util.PatchResources(b)
+			if err != nil {
+				return nil, ctrl.Result{}, err
+			}
+			resources = append(resources, flaggerResources...)
+		}
+	}
+
+	log.V(4).Info("wait for flagger helm release to be reconciled")
+	if !f.helmReleaseReady(ctx, fleet, resources) {
```
Comment (hzxuzhonghu) [inline pkg/fleet-manager/fleet_plugin_flagger.go:98]: duplicate with L68

Comment (hzxuzhonghu) [reply pkg/fleet-manager/fleet_plugin_flagger.go:98]: can you append the resources together?

File: pkg/fleet-manager/manifests/plugins/flagger.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+type: oci
+repo: oci://ghcr.io/fluxcd/charts
+name: flagger
+version: 1.x # update Flagger to the latest minor version
+targetNamespace: istio-system
```
Comment (hzxuzhonghu) [inline pkg/fleet-manager/manifests/plugins/flagger.yaml:5]: Why do you install flagger in istio-system

Comment (hzxuzhonghu) [reply pkg/fleet-manager/manifests/plugins/flagger.yaml:5]: Should have its own system namespace

**Review by hzxuzhonghu — APPROVED** [review]
/lgtm

**Pull request merged**